CHANGELOG.md

```diff
@@ -4,6 +4,8 @@ This file is used to list changes made in each version of the jenkins cookbook.
 
 ## Unreleased
 
+Update Jenkins apt/rpm repository key urls.
```

Comment:
The most recent changelog entries do not use bulleted lists. I followed that although earlier entries do. Was there an explicit style change or was this a formatting discrepancy with the recent entries?

Reply:
Those were done via automated PRs which don't know about the bullets. Please add a bullet here and try and fix the formatting of the others if you can.